packages/shared/src/lib/image.ts

```diff
@@ -258,6 +258,8 @@ export const cloudinaryIntegrationsSlackHeaderDark =
 
 export const cloudinaryTopReaderBadgeBackground =
   'https://media.daily.dev/image/upload/v1730888952/webapp/topReaderBadgeBackground.svg';
+export const cloudinaryPlusBackground =
+  'https://daily-now-res.cloudinary.com/image/upload/s--q2q8AL3Q--/f_auto/v1731397654/Upgrade_to_plus_i8phhr';
```

Comment:
Was this the correct image?

Reply:
No but don't want to block merge to intermediate for a image.

Reply:
👍